core/src/translate/mod.rs

```diff
@@ -281,20 +288,31 @@ pub fn translate(sql_statement: &SqlStatement) -> Result<Statement> {
 }
 
 pub fn translate_assignment(sql_assignment: &SqlAssignment) -> Result<Assignment> {
-    let SqlAssignment { id, value } = sql_assignment;
-
-    if id.len() > 1 {
-        return Err(
-            TranslateError::CompoundIdentOnUpdateNotSupported(sql_assignment.to_string()).into(),
-        );
-    }
+    let SqlAssignment { target, value } = sql_assignment;
+
+    let target = match target {
+        SqlAssignmentTarget::Tuple(_) => {
+            return Err(TranslateError::CompoundIdentOnUpdateNotSupported(
+                sql_assignment.to_string(),
+            )
+            .into());
+        }
+        SqlAssignmentTarget::ColumnName(SqlObjectName(targets)) => {
+            if targets.len() > 1 {
+                return Err(TranslateError::CompoundIdentOnUpdateNotSupported(
+                    sql_assignment.to_string(),
+                )
+                .into());
+            } else {
+                targets
+                    .first()
+                    .ok_or(TranslateError::UnreachableEmptyIdent)?
+            }
+        }
+    };
 
     Ok(Assignment {
-        id: id
-            .first()
-            .ok_or(TranslateError::UnreachableEmptyIdent)?
-            .value
-            .to_owned(),
+        id: target.value.to_owned(),
         value: translate_expr(value)?,
     })
```

Comment:
here, it would be good to add a new error for indicating tuple assignment is not supported.
// translate/error.rs - to add TupleAssignmentOnUpdate .. 
<img width="758" alt="image" src="https://github.com/user-attachments/assets/e21e25f1-0dd5-4d94-8128-d61a207315d6">


then,
```rust
    let SqlAssignment { target, value } = sql_assignment;
    let id = match target {
        SqlAssignmentTarget::Tuple(_) => {
            return Err(TranslateError::TupleAssignmentOnUpdateNotSupported(
                sql_assignment.to_string(),
            )
            .into());
        }
        SqlAssignmentTarget::ColumnName(SqlObjectName(id)) => id,
    };

    if id.len() > 1 {
        return Err(
            TranslateError::CompoundIdentOnUpdateNotSupported(sql_assignment.to_string()).into(),
        );
    }

    Ok(Assignment {
        id: id
            .first()
            .ok_or(TranslateError::UnreachableEmptyIdent)?
            .value
            .to_owned(),
        value: translate_expr(value)?,
    })
```